backend/test/test_common/unit/utils/test_type_conversion_utils.py

```diff
@@ -213,5 +217,272 @@ def test__convert_pandas_series_to_numpy__int64_to_int32_with_nans_throws_error(
             convert_pandas_series_to_numpy(int_series, np.int32)
 
             self.assertIn(
-                "Cannot convert a pandas Series object to an integer dtype if it contains NaNs", logger.output[0]
+                "Cannot convert a pandas Series object to an integer dtype if it contains NA values.", logger.output[0]
             )
+
+
```

Comment:
Seems like all the test cases above this line are redundant given the parametrized test cases below. Any reason to keep them around?

Reply:
they are not fully redundant.  But I take your point and will clean it up.